### PR TITLE
upgrade swc to v0.37.0 (batch-2 widgets)

### PR DESCRIPTION
**Pull request by inasirhussain**
This PR takes care of upgrading **sp-card** to version **0.37.0** in swc-uxp-wrappers. Also we have introduced support of **sp-asset, sp-quick-actions** and **sp-meter**

Covering JIRA tickets -

https://jira.corp.adobe.com/browse/UXP-22042
https://jira.corp.adobe.com/browse/UXP-22183
https://jira.corp.adobe.com/browse/UXP-22185
https://jira.corp.adobe.com/browse/UXP-22186

### CODE REVIEW

File: packages/card/package.json [hunk 1]
```diff
@@ -23,8 +23,10 @@
         "./src/card.css.js": "./src/card.css.js"
     },
     "dependencies": {
-        "@swc-uxp-internal/card": "npm:@spectrum-web-components/card@0.13.4",
-        "@swc-uxp-wrappers/checkbox": "1.0.0"
+        "@swc-uxp-internal/card": "npm:@spectrum-web-components/card@0.37.0",
+        "@swc-uxp-wrappers/checkbox": "1.0.0",
```
Comment (bhatiaPankaj) [inline packages/card/package.json:27]: Version of elementary wrappers must be updated (2.0.0)

Comment (inasirhussain) [reply packages/card/package.json:27]: As discussed earlier, this version does not yet. So lets not put it right now.
once we bump up the package versions during that stage we should update the transitive dependencies as well.

File: packages/meter/src/uxp-meter.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* write uxp style overrides */
+
+/*
+Copyright 2023 Adobe. All rights reserved.
```
Comment (deepthi-majji) [inline packages/meter/src/uxp-meter.css:16]: comments are duplicated.

File: packages/card/src/uxp-card.css [hunk 3]
```diff
@@ -11,3 +11,145 @@ governing permissions and limitations under the License.
 */
 
 /* write uxp style overrides */
+
+:host {
```
Comment (deepthi-majji) [inline packages/card/src/uxp-card.css:15]: @inasirhussain I want to understand each of the style overrides in this file. Do you have tracking tickets or wiki?

Comment (inasirhussain) [reply packages/card/src/uxp-card.css:15]: @deepthi-majji - This all overrides are to polyfill the various missing logical properties. I can do one thing, put the UXP JIRA ticket tracking the logical property support.

File: packages/meter/src/uxp-meter.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* write uxp style overrides */
+
+:host {
+    max-width: var(
+        --mod-progressbar-max-size,
```
[the remark this answers is not in the document]
Comment (inasirhussain) [reply packages/meter/src/uxp-meter.css:19]: Taking care by adding tracking UXP jira ticket for logical property support in UXP

**Review by deepthi-majji — APPROVED** [review]
LGTM.

**Pull request merged**